### ISSUE TITLE
Make startpage parameter more robust

### ISSUE THREAD

**Issue by ChristianMayer** [CometVisu/CometVisu#350]
When the startpage URL parameter is set but doesn't exists (e.g. due to a typo) the CV doesn't display the page.

Testcase: http://wiregate/cometvisu/?config=demo&startpage=id_51 (instead of the working http://wiregate/cometvisu/?config=demo&startpage=id_51_ )

This also creates a (sporadic?) error in the console:

```
templateengine.js [sm]:1575 Uncaught TypeError: Cannot read property 'beforePageChange' of undefined
TemplateEngine.scrollToPage @   templateengine.js [sm]:1575
setup_page  @   templateengine.js [sm]:1325
(anonymous function)    @   templateengine.js [sm]:216
execCb  @   require-2.1.15.min.js.src.js:29
```

**Comment by ChristianMayer** [comment]
This should be fixed for 0.10.0

**Comment by peuter** [comment]
Closed by #457
